autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -240,6 +281,24 @@ def on_epoch_end(self, X: Dict[str, Any], epoch: int) -> bool:
         If returns True, the training is stopped
 
         """
+        if X['is_cyclic_scheduler']:
+            if hasattr(self.scheduler, 'T_cur') and self.scheduler.T_cur == 0 and epoch != 1:
+                if self.use_swa:
+                    self.swa_model.update_parameters(self.model)
+                if self.use_se:
```

Comment:
Can you improve the name of use_se? 
In case there is a new person working, it will find it hard to understand what this is. Also, it is nice is you could also change the name `use_swa` to something more intuitive, it doesn't matter that the name is a bit longer

Reply:
okay I'll use the full names then. I also dont like this way of naming :P